homeworks/13/instagram-api/instagram-api/src/main/java/elsys/instagram/api/RegisterServlet.java

```diff
@@ -0,0 +1,40 @@
+package elsys.instagram.api;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.*;
+
+import com.google.gson.JsonObject;
+
+
+@WebServlet(name="RegisterServlet", urlPatterns="/register")
+public class RegisterServlet extends HttpServlet {
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        String first_name = req.getParameter("first_name");
+        String last_name = req.getParameter("last_name");
+        String username = req.getParameter("username");
+        String password = req.getParameter("password");
```

Comment:
1. Request should be done in JSON format
2. Would've been better if error handling was applied to this logic.